app/models/library_entry.rb

```diff
@@ -99,9 +99,15 @@ def activity
   end
 
   before_save do
-    if status_changed? && status == :completed && media.progress_cap
+    if status_changed? && status == 'completed' && media.progress_limit
```

Comment:
nvm you were right I thought it would need to know the column name to reference (ie: status) but I guess it is smarter than me 😏